packages/atomizer/src/rules.js

```diff
@@ -2867,6 +2867,71 @@ module.exports = [
     },
     /**
     ==================================================================
+    PLACE-CONTENT
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Place content',
+        matcher: 'Pc',
+        allowParamToValue: false,
+        styles: {
+            'place-content': '$0 $1',
+        },
+        arguments: [
+            sortKeys({
+                ...baselinePosition,
+                ...contentDistribution,
+                ...contentPosition,
+            }),
+            sortKeys({
+                ...baselinePosition,
+                ...contentDistribution,
+                ...contentPosition,
+            }),
+        ],
+    },
+    /**
+    ==================================================================
+    PLACE-ITEMS
+    ==================================================================
+    */
+    {
+        type: 'pattern',
+        name: 'Place items',
+        matcher: 'Pi',
+        allowParamToValue: false,
+        styles: {
+            'place-items': '$0 $1',
+        },
+        arguments: [sortKeys({ ...alignItems }), sortKeys({ ...alignItems })],
```

Comment:
https://www.w3.org/TR/css-align-3/#place-items-property

[<'align-items'>](https://www.w3.org/TR/css-align-3/#propdef-align-items) [<'justify-items'>](https://www.w3.org/TR/css-align-3/#propdef-justify-items)[?](https://www.w3.org/TR/css-values-4/#mult-opt)

Reply:
good point, didn't realize the values were different, obvious now since its shorthand for other properties. fixed now.